actioncable/lib/action_cable/channel/base.rb

```diff
@@ -253,7 +253,7 @@ def processable_action?(action)
         def dispatch_action(action, data)
           logger.info action_signature(action, data)
 
-          if method(action).arity == 1
+          if method(action).arity.abs == 1
```

Comment:
I flipped the condition (silly me) but I'm struggling a bit with adding the test (mainly because I can't get it to run just this one test).